cmd/node/config/enableEpochs.toml

```diff
@@ -190,6 +190,9 @@
         { EpochEnable = 1, MaxNumNodes = 56, NodesToShufflePerShard = 2 }
     ]
 
+    # HeartbeatDisableEpoch represents the epoch when heartbeat v1 messages stop being sent and processed
+    HeartbeatDisableEpoch = 1
```

Comment:
Please set this to 2, otherwise we risk breaking the network in epoch 1 when we do the upgrade from the "base branch" to the hb v2 feature branch